agent/php_error.h

```diff
@@ -137,6 +137,7 @@ extern void nr_php_error_install_exception_handler(TSRMLS_D);
 extern nr_status_t nr_php_error_record_exception(nrtxn_t* txn,
                                                  zval* exception,
                                                  int priority,
+                                                 bool add_to_segment,
```

Comment:
Please update doc string with explanation of the new parameter.

Reply:
updated in [bce3e14](https://github.com/newrelic/newrelic-php-agent/pull/767/commits/bce3e1462db26bb5f841542c199dae92ba96a1b3)